meteor/server/api/rundown.ts

```diff
@@ -158,77 +262,66 @@ export function afterRemoveParts (rundownId: string, removedParts: DBPart[]) {
  * Adlib/dynamic parts get assigned ranks based on the rank of what they are told to be after
  * @param rundownId
  */
-export function updatePartRanks (rundownId: string): Array<Part> {
-	const allSegments = Segments.find({ rundownId: rundownId }, { sort: { _rank: 1 } }).fetch()
-	const allParts = Parts.find({ rundownId: rundownId }, { sort: { _rank: 1 } }).fetch()
+export function updatePartRanks (rundown: Rundown): Array<Part> {
+	// TODO-PartInstance this will need to consider partInstances that have no backing part at some point, or do we not care about their rank?
```

Comment:
In case you arent aware, all the `TODO-PartInstance` comments are things to fix in a later phase of PartInstances. 
Im not sure if thats enough. because then we are likely to get an instance with the same rank as a real part. and so how should the ui draw that, and what should the next be?
This is part of a bigger discussion and task for handling instances without the original parts